chatlearn/models/base_module.py

```diff
@@ -767,18 +790,39 @@ def broadcast_parameter(self, rank, src_rank, group_name, pipe_stage=0):
         :meta private:
         """
         tensors = []
+        # breakpoint()
+        print(f"debug from {src_rank} to {rank} self._parameters_to_sync[{pipe_stage}]: {len(self._parameters_to_sync[pipe_stage])} {self._parameters_to_sync[pipe_stage][0][0]}", flush=True)
```

Comment:
a debug level msg should not be printed by `print`

Reply:
Delete already.